test/ctia/test_helpers/crud.clj

```diff
@@ -76,46 +78,52 @@
                           "when wait_for is not specified"))))
 
     (testing "testing wait_for values on entity update / patch"
-      (let [entity-id (-> (format "ctia/%s?wait_for=true" entity)
-                          (post :body new-record
+      (let [entity-id (-> (POST app
+                                (format "ctia/%s?wait_for=true" entity)
+                                :body new-record
                                 :headers headers)
                           :parsed-body
                           entity->short-id)
```

Comment:
Below I changed `method` to `method-kw` to avoid pointer equality checks (ie., `PUT` => `:PUT`).